tensorzero-core/src/db/clickhouse/evaluation_queries.rs

```diff
@@ -100,13 +100,13 @@ fn get_evaluation_result_datapoint_id_subquery(
     let query = format!(
         "all_inference_ids AS (
             SELECT DISTINCT inference_id
-            FROM TagInference FINAL WHERE key = 'tensorzero::evaluation_run_id'
```

Comment:
needed to change these because it was massively dragging performance.  `select distinct` will take care of the deduplication, and Chat/JsonInference tables don't get much updates, so this should be relatively safe while making performance much better.